### PR TITLE
[incubator/druid] modify the default value 

### LINKED ISSUE #23071: [incubator/druid] change druid_coordinator_balancer_strategy
**Is your feature request related to a problem? Please describe.**
we deployed druid cluster on k8s. It works off-line, and data ingest once by day. coordinator is always moving segments around to keep the cluster well-balanced

**Describe the solution you'd like**
change coordinator default config:  druid_coordinator_balancer_strategy: cost

**Describe alternatives you've considered**
druid default balancer strategy is cost, and I suspect that cachingCost have some issues

**Additional context**
no
#23072

### PR DESCRIPTION
**Pull request by camelus0211**
no
#### What this PR does / why we need it:
change coordinator default config: druid_coordinator_balancer_strategy: cost

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #23071

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: incubator/druid/values.yaml [hunk 1]
```diff
@@ -96,7 +96,7 @@ coordinator:
   config:
     DRUID_XMX: 256m
     DRUID_XMS: 256m
-    druid_coordinator_balancer_strategy: cachingCost
+    druid_coordinator_balancer_strategy: cost
```
Comment (AWaterColorPen) [inline incubator/druid/values.yaml]: It is better to remove the `druid_coordinator_balancer_strategy` setting

**Review by AWaterColorPen — APPROVED** [review]

File: incubator/druid/values.yaml (hunk 1 above)
Comment (camelus0211) [reply incubator/druid/values.yaml]: ok, thanks #23072

**Pull request closed without merging**